group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -24387,6 +24387,268 @@ public void testShareGroupRecordsNoExceptionOnReplay(CoordinatorRecord record) {
         assertDoesNotThrow(() -> context.replay(record));
     }
 
+    @Test
+    public void testConsumerGroupAssignmentResolvesWithCompaction() {
```

Comment:
How about
```suggestion
    public void testConsumerGroupPartitionEpochsWithCompaction() {
```
The same for streams.

Reply:
`testReplayConsumerGroupCurrentMemberAssignmentWithCompaction` is probably a better name.